src/MfaVerifyController.js

```diff
@@ -110,6 +110,11 @@ function (Okta, BaseLoginController, TOTPForm, YubikeyForm, SecurityQuestionForm
         this.listenTo(this.model, 'change:rememberDevice', function (model, rememberDevice) {
           model.get('backupFactor').set('rememberDevice', rememberDevice);
         });
+
+        // Listen to specific errors and bubble them up
+        this.listenTo(this.model, 'errors:verify', function (model, err) {
+          this.trigger('afterError', { controller: this.className }, err);
```

Comment:
feels like we shall build abstract layer (util function) for `this.trigger('afterError',...` to be a util function so that we only need to maintain the data structure for `afterError` at one place.

Reply:
We do! We have `Util.triggerAfterError` that will account for some of the defaults. I'll make sure this gets updated.